pvacseq/pvacseq.py

```diff
@@ -71,6 +71,11 @@ def main():
     config_files_parser.set_defaults(func=lib.config_files)
 
     parser.add_argument(
+        "vaccine_design",
+        help="Generate optimal vaccine configuration from protein fasta.",
```

Comment:
"optimal" is a loaded word. Maybe change to something doesn't necessarily imply that you're guaranteed to find the absolute best solution (because I don't believe that this algorithm does that).  Perhaps this?: "Generate an optimized ordering for peptides in a DNA vaccine"